mflike/mflike.py

```diff
@@ -357,7 +359,14 @@ def _get_power_spectra(self, cl, **params_values_nocosmo):
             p = m["pol"]
             i = m["ids"]
             w = m["bpw"].weight.T
-            clt = w @ DlsObs[p, m["t1"], m["t2"]]
+            dls_obs = DlsObs[p, m["t1"], m["t2"]]
+            # If symmetrize = False, the (ET, exp1, exp2) spectrum 
+            # will have the flag m["hasYX_xsp"] = True. 
+            # In this case, the power spectrum
+            # is computed as DlsObs["te", m["t2"], m["t1"]], to associate
+            # T --> exp2, E --> exp1
+            if m["hasYX_xsp"]: dls_obs = DlsObs[p, m["t2"], m["t1"]]
```

Comment:
Here, while keeping the comments above, you can write
```python
dls_obs = DlsObs[p, m["t2"], m["t1"]] if m["hasYX_xsp"] else DlsObs[p, m["t1"], m["t2"]]
``` 
in such a way reader can associate the comment to both conditions